rewrite-java/src/main/java/org/openrewrite/java/tree/Javadoc.java

```diff
@@ -292,18 +292,43 @@ public String printComment() {
      */
     @Value
     @EqualsAndHashCode(callSuper = false, onlyExplicitlyIncluded = true)
-    @With
     class LineBreak implements Javadoc {
         @EqualsAndHashCode.Include
         UUID id;
 
         String margin;
+        EndOfLine endOfLine;
         Markers markers;
 
         @Override
         public <P> Javadoc acceptJavadoc(JavadocVisitor<P> v, P p) {
             return v.visitLineBreak(this, p);
         }
+
+        public enum EndOfLine {
```

Comment:
May use a boolean instead if that's preferred.
The new line characters are separated to simplify/clean up whitespace formatting elsewhere.